.editorconfig

```diff
@@ -36,3 +36,5 @@ dotnet_sort_system_directives_first = true
 
 # Expression-level preferences
 csharp_style_implicit_object_creation_when_type_is_apparent = true:warning
+# IDE0005: Using directive is unnecessary.
+dotnet_diagnostic.IDE0005.severity = warning
```

Comment:
Can we make this non-blocking? This means that you'll be forced to deal with errors while refactoring.

Reply:
Yup, I agree... I didn't think a warning would cause build errors. Fixed now :)